public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -112,11 +113,26 @@ const restructureLog = memoizeOne((line: string, prettifyLogMessage: boolean): s
 });
 
 class UnThemedLogRowMessage extends PureComponent<Props> {
+  logRowRef: React.RefObject<HTMLTableCellElement> = React.createRef();
+
   onContextToggle = (e: React.SyntheticEvent<HTMLElement>) => {
     e.stopPropagation();
     this.props.onToggleContext();
   };
 
+  onShowContextClick = (e: React.SyntheticEvent<HTMLElement, Event>) => {
+    this.onContextToggle(e);
+    if (this.props.scrollElement && this.logRowRef.current) {
+      this.props.scrollElement.scroll({
+        behavior: 'smooth',
+        top:
+          this.props.scrollElement.scrollTop +
+          this.logRowRef.current.getBoundingClientRect().top -
+          window.innerHeight / 2,
+      });
+    }
+  };
```

Comment:
```suggestion
  onShowContextClick = (e: React.SyntheticEvent<HTMLElement, Event>) => {
  const { scrollElement } = this.props;
    this.onContextToggle(e);
    if (scrollElement && this.logRowRef.current) {
      scrollElement.scroll({
        behavior: 'smooth',
        top:
          scrollElement.scrollTop +
          this.logRowRef.current.getBoundingClientRect().top -
          window.innerHeight / 2,
      });
    }
  };
```

This is just preference. 🙂

Reply:
Yep, done in https://github.com/grafana/grafana/pull/55989/commits/ec38566f2c2e48d6da9830e01f786e333a22160d